src/pages/events.tsx

```diff
@@ -0,0 +1,404 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable react/jsx-key */
+/* eslint-disable react/no-unescaped-entities */
+import React, { useEffect, useState } from "react";
+import Image from "next/image";
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import "./events_style.css";
```

Comment:
Could you move this to `src/styles`? Then, you could use it with `import @/styles/events.css;`.